### PR TITLE
allow to run non-root

### PR DESCRIPTION
**Pull request by dmio**
Hi!
I've added few changes to your script.
- add support for stop daemon
- support to run under non root user

### CODE REVIEW

File: init.d/autossh [hunk 1]
```diff
@@ -45,6 +45,13 @@ start() {
     eend $?
 }
 
+stop() {
```
Comment (dmoulding) [inline init.d/autossh:47]: Absent a reason why the default stop function (which calls start-stop-daemon for us) is insufficient, I'm inclined to continue to let it use the default, which shows this when I stop it:

dmoulding@hitch ~ $ sudo rc-service autossh.gate.danm.net stop
- Stopping autossh.gate.danm.net ...                                     [ ok ]

Comment (dmio) [reply init.d/autossh:47]: I checked this, and you are right. Default stop() works ok. I added it log time ago and don't know now why. :)
I can try to delete this commit on my side and it should disappear here.

Comment (dmoulding) [reply init.d/autossh:47]: I'll just cherry pick the other two commits.

**Comment by dmoulding** [comment]
Cherry-picked to master. Thanks for the contribution!

**Pull request closed without merging**